articles/openshift/oshift-ref-cluster-monitoring.md

```diff
@@ -24,7 +24,7 @@ OpenShift 3.11 ships with a pre-configured and self-updating monitoring stack ba
 
 ### Resource
 
-Cluster monitoring is deployed with persistent storage, meaning that the metrics are stored on a persistent volume and can survive a pod being restarted or recreated. By default, each Alertmanager instance is allocated 2GiB and each Prometheus instance is allocated 50GiB. In a standard cluster with two infrastructure nodes this will mean 106GiB of storage will be used for cluster monitoring. If the persistent volumes fill up, or you notice them getting close to being full, you can expand the persistent volumes by following the steps in [*How to expand OpenShift persistent volumes*](https://docs.ukcloud.com/articles/openshift/oshift-how-expand-persistent-vols.html) or by raising a Service Request in the UKCloud Portal.
+Cluster monitoring is deployed with persistent storage, meaning that the metrics are stored on a persistent volume and can survive a pod being restarted or recreated. By default, each of the three Alertmanager instances is allocated 2GiB and the two Prometheus instances are allocated 50GiB each. In a standard cluster with two infrastructure nodes this will mean 106GiB of storage will be used for cluster monitoring. If the persistent volumes fill up, or you notice them getting close to being full, you can expand the persistent volumes by following the steps in [*How to expand OpenShift persistent volumes*](https://docs.ukcloud.com/articles/openshift/oshift-how-expand-persistent-vols.html) or by raising a Service Request in the UKCloud Portal.
```

Comment:
For our v3 deployments 3 infrastructure nodes in now our standard deployment this doesn't appear to change the scale of monitoring pods deployed so storage use is the same, I think we should remove 'In a standard cluster with two infrastructure nodes' or at least change 'two' to 'three' there.

Reply:
Good point, I've removed the reference to the number of nodes in the latest commit.